src/main/webapp/index.html

```diff
@@ -7,20 +7,57 @@
    <script src="script.js"></script>
  </head>
  <body>
-   <div id="MainSection-container">
+   <div id="MainSection">
      <h1>Main Section</h1>
    </div>
  
-   <div id="TwitterBot-container">
-     <h1>Twitter Bot</h1>
-   </div>
+   <div id="TwitterBot">
+     <center><h1>Twitter Bot</h1></center>
+     <header>
+         <!-- It adds a profile picture -->
+    <img src=https://s3.amazonaws.com/caamedia/wp-content/uploads/2017/03/JivanAtmanBest-700x450.jpg alt="Eren Yeager's profile picture" class="profile-picture">
+  <div class="profile-name">
+    <!-- It adds a profile name and handle -->
+    <h3>Eren Yeager</h3>
+    <h4>@ErenYeager</h4>
+  </div>
+   <!-- It adds a follow button, which says 'Thank you for following me.' It does this when it is clicked on. -->
+  <div class="follow-button">
+     <button type="button" onclick="alert('Thank you for following me.')">Follow</button>
+  </div>
+</header>
+<div id="inner">
+  <!-- It adds a tweet with a time and date. -->
+  <p>The attacks on Asian-Americans are the most heartbreaking thing. These attacks need to be stopped. We need to continue to speak out against these inhumane actions because Asians are not a virus, but hate is.</p>
+  <span class="date">2:32 P.M. - 18 March 2021</span>
+  <hr>
+</div>
+<footer>
+<!-- It adds how many likes and retweets this person got for their post. -->
```

Comment:
these are good, but once again , all of this data needs to be retrieved from the backend.